### ISSUE TITLE
Cannot redeclare block-scoped variable 'ngDevMode'.

### ISSUE THREAD

**Issue by igormap42** [NMilicic/ngx-bootstrap-product-tour#14]
After every `npm install`  I'm getting an error: 

> node_modules/ngx-bootstrap-product-tour/node_modules/@angular/core/src/render3/ng_dev_mode.d.ts(9,11): error TS2451: Cannot redeclare block-scoped variable 'ngDevMode'.

If to delete `@angular` folder from `node_modules/ngx-bootstrap-product-tour/node_modules`, it's compiling. But still I need to delete it every time I'm installing smth. Is it possible to fix this error once and forever?

P.S. I'm using angular 5

**Comment by salmaarsh06** [comment]
remove node_modules from the package , it will work fine .

**Comment by igormap42** [comment]
@salmaarsh06 thank you for your interest. The problem is that, as I wrote before, if to delete @angular folder or node_modules (the same) it works, but temporary. Every new call of `npm install` will retrieve the problem.

**Comment by salmaarsh06** [comment]
yeah , i experienced the same problem , but thats the only solution that worked for me .

**Comment by patcat** [comment]
any updates on this one? I'd love to try to use this but can't use it if I need to set up a script every time to clear that `node_modules` folder within the module each time

**Comment by bobbyfine** [comment]
same problem here

**Comment by bartromgens** [comment]
I think the Angular dependencies should be moved to `peerDependencies` in `package.json` to solve this.

**Comment by swapnildubey-7** [comment]
Any update on this issue?
I am using Angular 7.

**Comment by afzalriz304** [comment]
downgrade ngx-boostrap to @2.0.5.

**Comment by nfgallimore** [comment]
This solution worked for me:

https://stackoverflow.com/a/52318129/3097454
